frontend/src/js/entity-history/actions.ts

```diff
@@ -73,12 +74,12 @@ export const loadHistoryData = createAsyncAction(
 )<
   void,
   {
-    currentEntityCsvUrl: string;
+    currentEntityCsvUrl: ResultUrlsWithLabel;
```

Comment:
I think this should remain a string as well (unless you see some benefit of adding the label information here? --> please explain)